lib/moby_derp/system_config.rb

```diff
@@ -17,6 +17,7 @@ def initialize(config_data_or_filename, moby_info, logger)
 				raise ArgumentError, "Unsupported type for config_data_or_filename parameter"
 			end
 
+			@host_hostname        = @config["host_hostname"] || Socket.gethostname
```

Comment:
Use of the `Socket` constant requires `require "socket"`.  There also doesn't appear to be a new test verifying that the default value of this system configuration parameter comes from `Socket.gethostname`.